src/CMakeLists.txt

```diff
@@ -142,7 +143,9 @@ set_target_properties(
 list(APPEND _compile_definitions VERSION="${PROJECT_NAME} ${PROJECT_VERSION} - Utilities Update")
 
 if (ENABLE_PHC)
-    target_precompile_headers(${PROJECT_NAME} PRIVATE phc.hxx)
+    if (NOT MSVC)
+        target_precompile_headers(${PROJECT_NAME} PRIVATE phc.hxx)
```

Comment:
Why is this excluded for MSVC in general? If it doesn't work with Visual Studio 2019 and below only, please exclude the specifc version with MSVC_VERSION. 
https://cmake.org/cmake/help/latest/variable/MSVC_VERSION.html

I don't have a newer visual studio version so i can't try it. Just a question